frontend/public/components/app.jsx

```diff
@@ -120,8 +121,8 @@ class App extends React.PureComponent {
             <LazyRoute path="/overview/all-namespaces" exact loader={() => import('./overview' /* webpackChunkName: "overview" */).then(m => m.OverviewPage)} />
             <Route path="/overview" exact component={NamespaceRedirect} />
 
-            <LazyRoute path="/catalog/all-namespaces" exact loader={() => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
-            <LazyRoute path="/catalog/ns/:ns" exact loader={() => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage)} />
+            <Route path="/catalog/all-namespaces" exact component={CatalogPage} />
+            <Route path="/catalog/ns/:ns" exact component={CatalogPage} />
```

Comment:
It looks like this increases our main bundle size by about 25%.

Does it work if you define loader once outside of `render` and keep `LazyRoute`?

```js
const catalogLoader = () => import('./catalog/catalog-page' /* webpackChunkName: "catalog" */).then(m => m.CatalogPage);
// ...
render() {
  // ...
  <LazyRoute path="/catalog/all-namespaces" exact loader={catalogLoader).then(m => m.CatalogPage)} />
```

Reply:
no, the full page still reloads

Reply:
I believe I found a way. I will send it to @dtaylor113